lua/spd/server/sv_spd.lua

```diff
@@ -43,7 +43,8 @@ local immuneEntities = {
     gmod_sent_vehicle_fphysics_base = true,
     gmod_sent_vehicle_fphysics_attachment = true,
     prop_vehicle_prisoner_pod = true,
-    phys_spring = true
+    phys_spring = true,
+    m9k_nervegasnade = true,
```

Comment:
```suggestion
    m9k_nervegasnade = true
```
Sorry, the next person will have to spend _hours_ longer to add a comma of their own when they need one